### ISSUE TITLE
Saving with an empty filename field results in unclear error

### ISSUE THREAD

**Issue by BigRoy** [Colorbleed/maya-capture-gui#16]
# Problem

With the version from #15 saving with an empty filename (when _save_ is True and _use default_ is False) in the  `IoPlugin` will result in the following error:

```python
# Traceback (most recent call last):
#   File "D:\git_pipeline\maya-capture-gui\capture_gui\app.py", line 372, in apply
#     options["filename"] = lib.capture_scene(options)
#   File "D:\git_pipeline\maya-capture-gui\capture_gui\lib.py", line 197, in capture_scene
#     path = capture.capture(**options)
#   File "P:\pipeline\dev\git\maya-capture\capture.py", line 176, in capture
#     **playblast_kwargs)
# RuntimeError: The file  exists and overwrite flag not set. Playblast aborted.
# 
```

# Proposal

It should be clearer to the end user what is going on. We could raise a clearer error message.

**Comment by BigRoy** [comment]
With https://github.com/Colorbleed/maya-capture-gui/pull/29/commits/84e764053c94a204931b9649799b9d36afed38e0 the `IoPlugin` will now convert an empty filename field to name the file `playblast` as default; this avoids the above error.

**Comment by BigRoy** [comment]
Fixed with #29
